src/script/client/ClientRepository.ts

```diff
@@ -287,10 +287,9 @@ export class ClientRepository {
    * @param password Password entered by user
    * @returns Resolves with the remaining user devices
    */
-  async deleteClient(clientId: string, password: string): Promise<ClientEntity[]> {
+  async deleteClient(clientId: string, password?: string): Promise<ClientEntity[]> {
     const selfUser = this.selfUser();
-    await this.clientService.deleteClient(clientId, password);
-    await this.deleteClientFromDb(selfUser.qualifiedId, clientId);
+    await this.core.service.client.deleteClient(clientId, password);
```

Comment:
This password can be undefined ?

Reply:
yes, that's on purpose. Temporary devices can be deleted without the password

Reply:
Okay, thanks!